tests/ipfwd/test_nhop_count.py

```diff
@@ -0,0 +1,178 @@
+import os
+import pytest
+import time
+import logging
+
+from collections import namedtuple
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer
+
+pytestmark = [
+    pytest.mark.topology('t1', 't2')
+]
+
+
+class NextHopGroup:
+    """
+    Create a list of next hop paths with given IP, MAC parameters
+    """
+    def __init__(
+        self, duthost, asic, count, iface, ip="172.16", mac="C0:FF:EE:00"
+    ):
+        IP_MAC = namedtuple('IP_MAC', 'ip mac')
+        self.iface = iface
+        self.arp_list = []
+        self.duthost = duthost
+        self.asic = asic
+        self.if_addr = "{}.0.3/16".format(ip)
+        self.ip = None
+        self.fileloc = os.path.join(os.path.sep, "tmp")
+        self.filename = os.path.join(self.fileloc, "static_arp.sh")
+
+        for i in range(11, count+11):
+            moff1 = "{0:x}".format(i/255)
+            moff2 = "{0:x}".format(i%255)
+
+            ipoff1 = i / 255
+            ipoff2 = i % 255
+            self.arp_list.append(IP_MAC(
+                "{}.{}.{}".format(ip, ipoff1, ipoff2),
+                "{}:{}:{}".format(mac, moff1, moff2)
+            ))
+
+        ip_iface = "ip address add {} dev {}".format(self.if_addr, self.iface)
+
+        logging.info("IF ADDR ADD {}".format(ip_iface))
+        # add IP address to the eth interface
+        result = asic.command(ip_iface)
+        pytest_assert(result["rc"] == 0, ip_iface)
+
+    def arps_add(self):
+        """
+        Create a file with static arp add commands, copy file
+        to DUT and run it from DUT
+        """
+        arp_cmd = "sudo {} arp -s {} {}"
+        with open(self.filename, "w") as fn:
+            for ip_mac in self.arp_list:
+                cmd = arp_cmd.format(self.asic.ns_arg, ip_mac.ip, ip_mac.mac)
+                fn.write(cmd + "\n")
+
+        self.duthost.copy(src=self.filename, dest=self.filename, mode=0755)
```

Comment:
can we use swssconfig to program arp via SONiC

Reply:
For the test case this should be fine. As with any ARP, neighbor orch will receive netlink events from kernel.